source/index.html.md

```diff
@@ -15,6 +15,7 @@ includes:
   - errors
 
 search: true
+#search_timeout: 300
```

Comment:
Instead of configuring per-page, maybe we can simplify and just make it a constant in `_search.js`?

Or if we want to get really crazy, we can run the search once immediately, test how long it seems to take, and then adjust the timeout accordingly if it takes too long. We may also be able to ask the search library how big the search corpus is and adjust search_timeout to be 0, 300, or 1000 based on that.